packages/core-data/src/types/README.md

```diff
@@ -7,7 +7,7 @@ The types in this directory are designed to support the following use-cases:
 * Provide type-hinting and documentation for entity records fetched in the various REST API contexts.
 * Type-check the values we use to *edit* entity records, the values that are sent back to the server as updates.
 
-**Warning:** The types model the _expected_ API responses which is **not** the same as having a full type safety for the API-related operations. The API responses are **not** used as-is and in many cases could disagree with the type definitions, for example a plugin could modify the response, or the API endpoint could have a nuanced implementation in which strings are sometimes used instead of numbers.
+**Warning:** The types model the _expected_ API responses which is **not** the same as having a Extensible type safety for the API-related operations. The API responses are **not** used as-is and in many cases could disagree with the type definitions, for example a plugin could modify the response, or the API endpoint could have a nuanced implementation in which strings are sometimes used instead of numbers.
```

Comment:
small nitpick: this change modifies the preceding `a` and it should be `an extensible`, both by adding the 'n' and un-capitalizing 'Extensible'